style.scss

```diff
@@ -164,8 +177,8 @@ main {
         display: flex;
         flex-direction: column;
         align-items: center;
-        padding: 1rem;
-        gap: 1rem;
+        padding: 16px;
```

Comment:
Since you use `16px` as an offset value quite often, please create a file named `offsets.scss` and define a variable `$default-offset` with this value there, and then use it. You can use `$default-offfset / 2` or `$default-offset * 3` in the code.